### DISCUSSION TITLE
Unminidifed Builds?

### DISCUSSION THREAD

**Discussion by prologic** [beercss/beercss#316] [Ideas]
Are you guys able to provide unminidifed versions of the builds? 🙏

**Comment by leonardorafael** [comment]
Hi, currently we have a `src` and `dist` folder on jsdelivr or npm:

https://www.jsdelivr.com/package/npm/beercss?tab=files
https://www.npmjs.com/package/beercss?activeTab=code

If this doesn't meet the requirements, I think we can add the unminified version on `dist` folder too.

**Reply by prologic** [reply]
Pleaee! 🙏

**Reply by leonardorafael** [reply]
Released in v3.5.8

Minified:
https://cdn.jsdelivr.net/npm/beercss@3.5.8/dist/cdn/beer.min.css
https://cdn.jsdelivr.net/npm/beercss@3.5.8/dist/cdn/beer.min.js

Unminified:
https://cdn.jsdelivr.net/npm/beercss@3.5.8/dist/cdn/beer.css
https://cdn.jsdelivr.net/npm/beercss@3.5.8/dist/cdn/beer.js

**Reply by prologic** [reply]
Thank you! 🙇‍♂️

**Comment by lighth7015** [comment]
Actually, this could be further enhanced with optional scss/sass source files!
